package.json

```diff
@@ -127,6 +135,10 @@
         {
           "command": "extension.debugJest",
           "group": "02_jest"
+        },
+        {
+          "command": "extension.watchJest",
+          "group": "02_jest"
```

Comment:
i dont want to flood the context menu with my extension thats why i would prefer to not add it to context menu.

Reply:
command palette and code lens should be enough

Reply:
ok fine :) i'll update it

Reply:
it's removed